libkbfs/journal_server.go

```diff
@@ -93,6 +98,22 @@ func makeJournalServer(
 	return &jServer
 }
 
+func (j *JournalServer) getDirLocked() string {
+	// TODO: This may end up making paths too long for some
+	// systems. We may end up having to drop the UID (since a
```

Comment:
The dir names are getting uncomfortably long -- the device ID is 70 characters, the UID is 32 characters, and the TLF ID is 32, plus the MD and block IDs. Any thoughts?